### ISSUE TITLE
Automation with Tasker

### ISSUE THREAD

**Issue by blubbsy** [chenxiaolong/DualBootPatcher#764]
Hi,

i'm loosing every time iam updating my LineageOS system the second system. For this reason i would like to use tasker to automatize my whole update process, including patching and flashing the newest update. Is it possible to automatize the process, e.g. via command line? This would be great, because i aready have in tasker programmed that it downloads the latest lineage and opengapps version, flashes it and installs supersu. adding the mbtool to do the task would make it complete :)

Blubbsy

**Comment by DeuShades** [comment]
Yes it is very possible to do this. At least it should be since DBP supports intents. I am trying to do the same thing, what I tried was running a script to send the intents (since my automation app was unable to correctly send the intent):

`am start -a com.github.chenxiaolong.dualbootpatcher.PATCH_FILE --es path /data/media/0/Download/LOS/LOS.zip --es rom_id data-slot-los`

This send the intent `com.github.chenxiaolong.dualbootpatcher.PATCH_FILE` with the extras `path=/data/media/0/Download/LOS/LOS.zip` and `rom_id=data-slot-los`, which is what [this manual](https://github.com/chenxiaolong/DualBootPatcher/blob/master/docs/INTENTS.md) tell us to do. However, this _just opens the patch-zip-activity of the app without patching anything_.

**How can we send the intent correctly?**

**Comment by DeuShades** [comment]
Hey, I don't want to be rude by stressing this, but can somebody _please_ help us here? Because having followed the manual, I have no idea what my mistake is.

**Comment by blubbsy** [comment]
i tried some stuff, but i'm facing always the same problem as you do. I only get to the screen to select the file to patch... thats quite useless :-(
